tests/util/test_color.py

```diff
@@ -57,3 +57,54 @@ def test_rgb_hex_to_rgb_list(self):
 
         self.assertEqual([51, 153, 255, 0],
                          color_util.rgb_hex_to_rgb_list('3399ff00'))
+
+
+class ColorTemperatureMiredToKelvinTests(unittest.TestCase):
+
+    def test_should_return_25000_kelvin_when_input_is_40_mired(self):
+        kelvin = color_util.color_temperature_mired_to_kelvin(40)
+        self.assertEqual(25000, kelvin)
+
+    def test_should_return_5000_kelvin_when_input_is_200_mired(self):
+        kelvin = color_util.color_temperature_mired_to_kelvin(200)
+        self.assertEqual(5000, kelvin)
+
+
+class ColorTemperatureKelvinToMiredTests(unittest.TestCase):
+
+    def test_should_return_40_mired_when_input_is_25000_kelvin(self):
+        mired = color_util.color_temperature_kelvin_to_mired(25000)
+        self.assertEqual(40, mired)
+
+    def test_should_return_200_mired_when_input_is_5000_kelvin(self):
+        mired = color_util.color_temperature_kelvin_to_mired(5000)
+        self.assertEqual(200, mired)
+
+
+class ColorTemperatureToRGB(unittest.TestCase):
```

Comment:
@balloob Farcy should comment one line lower so that, when I actually add the docstring, this comment can be auto-hidden.